openfecwebapp/config.py

```diff
@@ -15,7 +15,7 @@
 transition_url = env.get_credential('FEC_TRANSITION_URL', 'https://transition.fec.gov')
 classic_url = env.get_credential('FEC_CLASSIC_URL', 'http://classic.fec.gov')
 cms_url = env.get_credential('FEC_CMS_URL', '')
-canonical_base = 'https://beta.fec.gov'
+canonical_base = env.get_credential('CANONICAL_BASE', 'https://beta.fec.gov')
```

Comment:
My thought was that we'd set it in the env var after the DNS happens. But it could be the default if you think that's better.